docs/admin/setup/distributed.rst

```diff
@@ -502,3 +502,27 @@ It relays message between instances:
 
    .. figure:: distributed/registry.png
       :align: center
+
+
+Running multiple instances locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For testing purposes, you can also run multiple instances on the same machine. This can be done by using
+different ports and a unique instance ID for each instance.
+
+#. In ``application-dev.yml`` set ``eureka.client.enabled`` to ``true``.
```

Comment:
shouldn't this value rather be set in application-local.yml?

Reply:
You're right. Changed it.